<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="title">
   <string>Project selection</string>
  </property>
  <property name="subTitle">
   <string>Select a project to import.</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="filterLineEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="projectListWidget"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="restrainRemoteLinksCheckBox">
     <property name="text">
      <string>Show only projects which are remote links</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="restrainMaintainerCheckBox">
     <property name="text">
      <string>Show only projects on which I am maintainer</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="restrainBugownerCheckBox">
     <property name="text">
      <string>Show only projects on which I am bugowner</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
